src/content/docs/tutorials/contribution-counter.mdx

```diff
@@ -69,7 +69,7 @@ Use the `monetization` event to show a web monetized visitor how much they've se
 
 We’ll bind the `monetization` event if the visitor is web monetized (`window.MonetizationEvent` is defined).
 
-The `monetization` event contains details about the payments that occur. The `amountSent` attribute of the event returns the amount (`value`) and `currency` code of the sent payment. A currency code is a three-letter code, like USD, EUR, or GBP.
+The `monetization` event contains details about the payments that occur. The `amountSent` attribute of the event returns the amount (`value`) and `currency` code of the sent payment. A currency code is a three-letter code, like <abbr>USD</abbr> for US dollar, <abbr>EUR</abbr> for the euro, or <abbr>GBP</abbr> for Great Britain Pound.
```

Comment:
Missing `title`s in `abbr` here.

Reply:
I meant to remove those since they're defined in the text.